### ISSUE TITLE
Ubuntu: Unsupported distro 22.04

### ISSUE THREAD

**Issue by Gys** [cosandr/go-check-updates#1]
Hi!

Interesting package :-)

I tried this on my Ubuntu VM, but unfortunately it returns:
`time="2022-11-27T11:35:46Z" level=fatal msg="unsupported distro 22.04"`

Indeed the readme and the code do not refer to anything Debian. How unfortunate for me.

**Comment by cosandr** [comment]
Correct, I don't use Debian sorry. If that changes, I might add support for it, otherwise I'm willing to review a PR for such a feature.
